.github/ISSUE_TEMPLATE/new-repo-checklist.md

```diff
@@ -12,8 +12,9 @@ https://github.com/maplibre/[repo]
 **<...Explain why this repo is good for MapLibre project, its goals, and any other relevant info...>**
 
 ## Acceptance
-- [ ] Any two board members must agree to accept a new repository.
+- [ ] Any two board members must agree to accept a new repository, once the checklist is complete. _Movement to the `maplibre` organization may happen before or after the checklist is complete, but the two "Approved by" board members are required._
   **Approved by:** <@user1> <@user2>
+- [ ] Denote what tier of project it will be (`Core`, `Supported`, `Hosted`): <...tier...> (See discussion: https://github.com/maplibre/maplibre/discussions/132)
```

Comment:
Didn't we agree all project start as hosted? I'm not sure this is needed.
We can explain as part of this template how to get funds for a hosted project or simply link the relevant wiki page.

Reply:
I didn't think we had agreed to that, but it's a fair enough to default to Hosted and go from there.